src/routes/safe/components/Settings/SpendingLimit/utils.ts

```diff
@@ -0,0 +1,100 @@
+import GnosisSafeSol from '@gnosis.pm/safe-contracts/build/contracts/GnosisSafe.json'
+import { BigNumber } from 'bignumber.js'
+import generateBatchRequests from 'src/logic/contracts/generateBatchRequests'
+import { SENTINEL_ADDRESS } from 'src/logic/contracts/safeContracts'
+import { web3ReadOnly } from 'src/logic/wallets/getWeb3'
+import SpendingLimitModule from 'src/utils/AllowanceModule.json'
+import { SPENDING_LIMIT_MODULE_ADDRESS } from 'src/utils/constants'
+
+export const KEYCODES = {
+  TAB: 9,
+  SHIFT: 16,
+}
+
+export const fromTokenUnit = (amount: string, decimals: string | number): string =>
+  new BigNumber(amount).times(`1e-${decimals}`).toFixed()
+
+export const toTokenUnit = (amount: string, decimals: string | number): string =>
+  new BigNumber(amount).times(`1e${decimals}`).toFixed()
+
+export const currentMinutes = () => Math.floor(Date.now() / (1000 * 60))
```

Comment:
This function returns type `number`, isn't it?

Reply:
Yes, this was shamelessly copied from the Allowances contract's tests :grimacing: